lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -78,33 +80,36 @@ class ReportUIFeatures {
    * @param {LH.Result} report
    */
   initFeatures(report) {
-    if (this._dom.isDevTools()) return;
-
     this.json = report;
-    this._setupMediaQueryListeners();
-    this._setupExportButton();
-    this._setupThirdPartyFilter();
-    this._setUpCollapseDetailsAfterPrinting();
-    this._resetUIState();
-    this._document.addEventListener('keyup', this.onKeyUp);
-    this._document.addEventListener('copy', this.onCopy);
+
+    // Some features in the top right drop down menu don't work in the DevTools
+    // client. They could with some tweaks, but currently they don't. For example:
+    // Saving as HTML/JSON - does not bring up a file dialog, as one would expect in DevTools.
+    //                       also, it saves the AuditsPanel HTML, which is funky.
+    if (this._dom.isDevTools()) {
```

Comment:
what about just `display:none` on the export button? Then this stuff won't have to be special-cased

Reply:
done